pkg/apis/v1alpha1/workload.go

```diff
@@ -39,6 +39,7 @@ const (
 )
 
 // +kubebuilder:object:root=true
+// +kubebuilder:resource:categories="all"
```

Comment:
I think you may want to add that category to the appropriate cluster-scoped resources, too. (This will make them show up when listing resources in a namespace, too -- tested on k8s 1.21.)

I'm not sure whether you want to use the name "carto" or something more human-readable like "supplychain".

Reply:
Created an issue (#466) to capture this context so we can get this merged.